examples/plot_model_card.py

```diff
@@ -160,8 +165,8 @@
 model_card.add_table(
     folded=True,
     **{
-        "Hyperparameter search results": cv_results,
-        "Classification report": clf_report,
+        "Model description/Evaluation Results/Hyperparameter search results": cv_results,
+        "Model description/Evaluation Results/Classification report": clf_report,
```

Comment:
Okay, so it seems this template has a different structure to what is used in the [skops template](https://raw.githubusercontent.com/skops-dev/skops/main/skops/card/default_template.md).

E.g. metrics go into

`# Evaluation > ## Testing Data, Factors & Metrics > ### Metrics`

in the hub template, and into

`# Model description > ## Evaluation Results`

in the skops template. I'm impartial as to how to structure the default template, in this implementation I tried to  keep it the same as the previous skops model card.

Reply:
If we're changing it, I'd make it the same as the template we have in `huggingface_hub`

Reply:
Happy to change it to conform to the hub card. Since the skops default template was already there before I joined, I just assumed it was consciously changed to better fit the needs for sklearn models